### ISSUE TITLE
Can't import dashbaords with CLI

### ISSUE THREAD

**Issue by xavier-GitHub76** [apache/superset#28692]
### Bug description

Hello,

I try to export/import dashboard with cli but I have an error "superset.commands.exceptions.CommandInvalidError: Error importing dashboard"

### How to reproduce the bug

I export my dashboards with CLI : 
![image](https://github.com/apache/superset/assets/82046143/18150d8d-7492-4d65-8b61-ac8b87165878)

I delete all elements on my superset
![image](https://github.com/apache/superset/assets/82046143/b2716073-dedc-47d5-8a20-4c5d7cd8e440)
![image](https://github.com/apache/superset/assets/82046143/8ca72b56-63b0-4809-b2f6-427628b82c45)
![image](https://github.com/apache/superset/assets/82046143/613ed410-9092-4925-bb25-e6dd06529723)
![image](https://github.com/apache/superset/assets/82046143/5c807a3d-6f0b-42d2-8ead-1581cfb6a717)

I try import with cli
superset import-dashboards -p myDashboards.zip -u xavier

![image](https://github.com/apache/superset/assets/82046143/4a65057c-6201-43ef-abea-b4fa10d4d2f9)

I try also after a creation of database connexion : 
![image](https://github.com/apache/superset/assets/82046143/c8b9fe1f-beee-460d-9fc9-059e72b45c0b)

![image](https://github.com/apache/superset/assets/82046143/04f9acec-463d-420d-a30f-4b38a358c2ee)

Best regards

### Superset version

master / latest-dev

### Python version

3.9

### Node version

16

### Browser

Chrome

### Checklist

- [ ] I have searched Superset docs and Slack and didn't find a solution to my problem.
- [ ] I have searched the GitHub issue tracker and didn't find a similar bug report.
- [ ] I have checked Superset's logs for errors and if I found a relevant Python stacktrace, I included it here as text in the "additional context" section.

**Comment by anhvh1408** [comment]
I have same issue as you too, in my cases:
- I have dashboards in json file first (exported from superset 2.0.0)
- import json file to superset 3.1.3-dev -> work, dashboard shown perfectly
- then I export these dashboards to zip and import again (both in superset 3.1.3-dev) -> some work, some throw errors same as yours

**Comment by anhvh1408** [comment]
Okay, please follow this https://github.com/apache/superset/issues/26130 to debug your zip file and yaml file to see any arguments have errors

**Comment by xavier-GitHub76** [comment]
Hello,
my zip file is correct :  the zip file (created from CLI) is imported with success from gui
I have just the error when I import the zip file from CLI

I have try again on superset 4.0.0, I have the same behavior

Best regards

**Comment by maximStafeevAtAllscripts** [comment]
Yes, I have the same problem. Neither export nor import works on 4.0.1

**Comment by maximStafeevAtAllscripts** [comment]
If I import dashboard via GUI than import_dashboard from CLI successfully updates the dashboard.
I'm also wondering where should I place the password for the database when importing via CLI? Superset prompts you for it when importing through the GUI.

**Comment by muralikrishnankh** [comment]
@maximStafeevAtAllscripts What worked for me:
1. Unzip the file
2. Go to dashboards directory
3. Password is represented as XXXXXXXXX there
4. I replaced it with actual password and compressed again.
5. Upload it using CLI or GUI. Both works and doesn't ask for password prompt.

**Comment by rusackas** [comment]
Pinging @betodealmeida in case they have any insight here.

**Comment by Adriellevy** [comment]
> [@maximStafeevAtAllscripts](https://github.com/maximStafeevAtAllscripts) What worked for me:
> 
> 1. Unzip the file
> 2. Go to dashboards directory
> 3. Password is represented as XXXXXXXXX there
> 4. I replaced it with actual password and compressed again.
> 5. Upload it using CLI or GUI. Both works and doesn't ask for password prompt.

This stil works

**Comment by rusackas** [comment]
This has been silent for quite a while, so I'll assume the above solution worked for everyone. If anyone wants to add to the docs, or feels differently and would like this reopened/revisited, say the word.

**Comment by xavier-GitHub76** [comment]
I redo a test with 5.0.0rc4-dev : export / import are OK
